src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -32,51 +34,70 @@ Message bundles cannot contain arrays. Instead a namespace should be used to gro
 
-### PrefsEditorLoader
+### MsgLookup
 
-All versions of preferences editors (separated panel, full page with preview and full page without preview) are instances of a "PrefsEditorLoader" component. The PrefsEditorLoader coordinates the work of its three subcomponents: MessageLoader, TemplateLoader and PrefsEditor. In particular, the PrefsEditorLoader
+The `fluid.prefs.msgLookup` grade is required for any component through which a message lookup is performed. The msgLookup grade wires up the `msgLookup` IoC reference and the `lookup` method for retrieving messages from the Message Bundle (See: [Using Message Bundles](#using-message-bundles)). The msgLookup's message resolution is mediated by a `fluid.messageResolver` subcomponent and it is to this subcomponent that the `messageBase` option, containing the fetched Message Bundle, is set.
 
-- parses and assembles JSON strings loaded by the MessageLoader,
-- runs the assembled JSON through the [message resolver](http://wiki.fluidproject.org/display/docs/fluid.messageResolver) to create the lookup function, and
-- attaches the message resolver bundle as a member, accessible through `that.msgResolver`.
+For a `fluid.prefs.panel` component all of the `fluid.prefs.msgLookup` configuration is prewired. For other components to make use of this functionality the following configuration is needed:
 
-To access the message bundle from other components on the IoC tree, use `{prefsEditorLoader}.msgResolver`.
+- add `fluid.prefs.msgLookup` as a [parent grade](ComponentGrades.md#specifying-parent-grades)
+- add an instance of `fluid.messageResolver` as a subcomponent with the member name `msgResolver`
+- link the messageBase to the fetched Message Bundle.
+    - e.g. using an IoC reference to the [messageLoader](#messageloader): `{messageLoader}.resources.<messageBundle>.resourceText`
+- for an instance of `fluid.rendererComponent` it is also necessary to use the messageResolver's `resolve` method as the `messageLocator`
 
-### PrefsEditor
-
-PrefsEditor is the host component that holds all the actual panel (or adjuster) components as subcomponents. By default, the message bundle is *not* passed down to PrefsEditor. If your PrefsEditor component will need direct access to the message bundle, provide it at the instantiation of any PrefsEditor instance, as shown in the following example:
+#### Adding MsgLookup to a Component
```

Comment:
I don't have a good idea for this. Probably still use "Adding MsgLookup to a component" but add a note in the section that indicates renderer components need special handling and this approach is for other component types?